app-emulation/dosemu/files/dosemu-1.4.1_pre20130107-gcc-15.patch

```diff
@@ -0,0 +1,29 @@
+Bug: https://bugs.gentoo.org/944235
+--- a/src/arch/linux/dosext/sound/midid/midid.h
++++ b/src/arch/linux/dosext/sound/midid/midid.h
+@@ -8,7 +8,12 @@
+ #define _MIDID_H
+ 
+ typedef unsigned char byte;
++#if __STDC_VERSION__ < 202311L
+ typedef enum {FALSE, TRUE} bool;
++#else
++#define FALSE 0
++#define TRUE  1
++#endif
```

Comment:
How about using an anonymous enum instead?  And no need for checking `STDC_VERSION`, just include `<stdbool.h>`
```
#include <stdbool.h>
enum {FALSE, TRUE};
```

Reply:
Yes, it works as well, updated.